docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,62 @@
+# Getting started with supercomputing
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computations! This page is intended to give you some idea of what
+to expect from your journey into the exciting world of high-performance
+computing.
+
+## Which supercomputer to use?
+
+Projects may have access to multiple supercomputers. New users are recommended
+to start working on the Puhti supercomputer, which caters to a wide range of
+use cases. In time, you may find that you need more extensive parallelization
+options, in which case you may wish to run jobs on Mahti.
+
+## Which interface to use?
+
+### Web interface
+
+There are multiple ways to run computations on CSC supercomputers. The Puhti
+and Mahti web interfaces offer a beginner-friendly environment for working with
+supercomputers. The web interface includes various interactive applications like
+Jupyter, RStudio and Virtual Studio Code, which are useful for developing code
+and exploring data.
+
+### Command-line interface
+
+Because of the considerable scale of both data and processing involved in
+HPC, you might rather quickly find yourself in need of tools that are more
+efficient and powerful than those offered by the graphical web interface. 
+Effective use of supercomputers is based on running Unix commands in a
+text-based interface. Learning to interact with CSC devices in this way
+is a crucial step towards serious supercomputing.
+
+For Linux and macOS, the most natural way of using command-line tools
+is by opening a terminal and creating a remote SSH connection to a
+supercomputer. Windows users do not have access to a Unix-like terminal, so the
+recommended approach is using the login shell available in the supercomputer web
+interface.
+
+## General guidelines
+
+### Developing code
```

Comment:
Should here the first step be to develop on own computer which after all in many cases is the most convenient? It could then list reasons to move to away from that, ie need for GPU, need for more memory/CPU for full scale tests etc?

Reply:
Good point! I did manage to squeeze that in :)